api/src/test/java/org/openmrs/validator/ConditionValidatorTest.java

```diff
@@ -61,7 +67,10 @@ public void shouldFailIfGivenInstanceOfClassOtherThanCondition(){
 
 	@Test
 	public void shouldPassIfConditionClassIsPassed(){
-		validator.validate(new Condition(), errors);
+		Condition condition = new Condition();
+		condition.setCondition(new CodedOrFreeText(new Concept(), new ConceptName("name", new Locale("en")), "nonCoded"));
+		condition.setClinicalStatus(ConditionClinicalStatus.ACTIVE);
+		validator.validate(condition, errors);
 
 		Assert.assertFalse(errors.hasErrors());
 		Assert.assertFalse(errors.hasFieldErrors("condition"));
```

Comment:
The second assertion covers the first, making the first one redundant

Reply:
Yes true. Done

Reply:
Corrected